lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -33,10 +35,16 @@ def drawing(task_info=task)
         {}.tap do |new_anno|
           new_anno['task'] = @current['task']
           new_anno['task_label'] = task_label(task_info)
-          value_with_tool = (@current["value"] || []).map do |drawn_item|
-            tool_label = tool_label(task_info, drawn_item["tool"])
-            drawn_item.merge("tool_label" => tool_label)
+
+          value_with_tool = begin
+            Array.wrap(@current["value"]).map do |drawn_item|
+              tool_label = tool_label(task_info, drawn_item)
+              drawn_item.merge("tool_label" => tool_label)
+            end
+          rescue ToolFormattingError => e
+            @current["value"]
```

Comment:
Mostly I was wondering if we're not going to use exceptions for that purpose, we should probably not use them for flow controlish things either, and rather just have tool_label return nil in case of incompatible config style.